src/cpp/include/openvino/genai/streamer_base.hpp

```diff
@@ -39,7 +39,7 @@ class OPENVINO_GENAI_EXPORTS StreamerBase {
     /// @brief end is called at the end of generation. It can be used to flush cache if your own streamer has one
     virtual void end() = 0;
 
-    virtual ~StreamerBase();
+    virtual ~StreamerBase() = default;
```

Comment:
please, keep in .cpp, otherwise we will have issue with C++ RTTI (as class become header only)

Reply:
Sure, I'll keep it in .cpp file. I'm trying to fix win build. It fails due to this dtor: https://github.com/openvinotoolkit/openvino.genai/actions/runs/13241277766/job/36957252871#step:7:3726
Something with gmock. Investigating